common/src/main/java/com/tencent/rss/common/RemoteStorageInfo.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.common;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.common.base.Objects;
+import com.google.common.collect.Maps;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import com.tencent.rss.common.util.Constants;
+
+public class RemoteStorageInfo implements Serializable {
```

Comment:
Why don‘t use protobuf?

Reply:
It may not be appropriate to use protobuf class directly in upper bussiness layer.

Reply:
I think we should unify the serialization format.

Reply:
Why do this class need serialization?

Reply:
Because it is part of ShuffleHandle.

Reply:
That's OK